department-of-veterans-affairs/caseflow
# Fixing ama queue test that's failing

### Description
The AMA Queue spec started failing non-deterministically. It sometimes fails when we check to ensure the "New File" icon has disappeared after viewing the case in Reader. I believe this was happening because we clicked on the Reader link and then immediately jumped back to Queue. It's possible that the server didn't have a chance to update the case as viewed within this time, and therefore still thought there were new documents.

While deubgging I realized that Reader wasn't even loading properly because the documents we added didn't have document types and the user didn't have proper permissions. Once we fixed these problems, I added a check for text on the Reader screen, to ensure the test gives enough time before flipping back to Queue to check for the new documents icon.

This has the added benefit of ensure Reader loads from AMA Queues.

### Testing Plan
1. Run tests with ensure stable before change and see some failures.
1. Run tests with ensure stable after changes and see all successes.
